### ISSUE TITLE
Update metadata pinata.hashMetadata not working 

### ISSUE THREAD

**Issue by ateeqrehmani** [PinataCloud/Pinata-SDK#117]
unable to update metadata using Pinata SDK version 1.1.26

const options = {
          pinataMetadata: {
            keyvalues: {
              image: 'new url',
            }
          },
          pinataOptions: {
            cidVersion: 0
          }
        }

await pinata.hashMetadata('Qmdbm7r9VAcy2jmntA8MTdta6GZb3jKB59vrkdwkzz7Bv7', options)

![image (1)](https://user-images.githubusercontent.com/50024356/188722018-e40e9137-dcb3-402b-8d3c-2d4bdeb06d91.png)

always returning old metadata though providing new keyvalue pair.

**Comment by polluterofminds** [comment]
@ateeqrehmani sorry for the delay here. We are spinning up a new team dedicated to our open source tools. These issues are being tackled by that new team, so stay tuned.

**Comment by ateeqrehmani** [comment]
@rommel-pinata thanks for the reply
actually I used the same structure, have a look at attached screenshot. i have already saved my metadata, in metadata, there is a field called "image" with an empty value. now I wanted to update so I provided only key/value pair which needs to be updated
const options = {
     pinataMetadata: {
       keyvalues: {
         image: 'new url',
      }
},

what is your point do I need to provide a full metadata object with existing values which i don't want to update?
